### PR TITLE
New blankslate component

### PR DESCRIPTION
**Pull request by wfortin**
Small demo http://codepen.io/wfortin/pen/yONMMK?editors=1100

image

### CODE REVIEW

**Comment by GermainBergeron** [comment]
:shipit:

File: scss/typography/headings.scss [hunk 1]
```diff
@@ -1,6 +1,6 @@
 h1, h2, h3, h4, h5, h6 {
+  color: $medium-blue;
```
Comment (JPLachance) [inline scss/typography/headings.scss]: I'm not so sure it's a good idea to make all titles always blue by default. In the new modal, titles are white and we are gonna have overrides. I prefer to use "color classes" where we need them.

Comment (wfortin) [inline scss/typography/headings.scss]: Good point this should not have made it to the PR, I was experimenting.

Removing it.

**Comment by JPLachance** [comment]
:ship: :it: !

**Pull request merged**